lib/server/controllers/logs.controller.js

```diff
@@ -320,6 +320,44 @@ const getRootLogsHandler = async (request, response) => {
     }
 };
 
+/**
+ * Get all child logs by parent id
+ *
+ * @param {Object} request The *request* object represents the HTTP request and has properties for the request query
+ *                         string, parameters, body, HTTP headers, and so on.
+ * @param {Object} response The *response* object represents the HTTP response that an Express app sends when it gets an
+ *                          HTTP request.
+ * @returns {undefined}
+ */
+const getChildLogsByParentIdHandler = async (request, response) => {
+    const requestDto = await dtoValidator(GetLogDto, request, response);
+
+    if (requestDto) {
+        try {
+            const { params: { logId } } = requestDto;
+            const { count, childLogs } = await logService.getChildLogsByParentId(logId);
+
+            if (count > 0) {
```

Comment:
Why would you throw if count is 0? In this case, you should simply return an empty, which is normal.

Reply:
Yes I would say so